src/tail/tail.rs

```diff
@@ -85,8 +82,8 @@ pub fn uumain(args: Vec<String>) -> i32 {
 
     match given_options.opt_str("n") {
         Some(n) => {
-            let mut slice = n.as_slice();
-            if slice.len() > 0 && slice.char_at(0) == '+' {
+            let mut slice: &str = n.as_ref();
+            if slice.len() > 0 && slice.chars().next().unwrap_or('_') == '+' {
```

Comment:
For all of these `unwrap_or` things, I think you should either change `unwrap_or` to just `unwrap` or remove the length check.